### PR TITLE
Update Help Window

### PR DESCRIPTION
**Pull request by WM71811**

### CODE REVIEW

**Review by Persdre — DISMISSED** [review]
LGTM

File: src/main/java/seedu/address/model/ingredient/UniqueIngredientList.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public void setIngredients(List<Ingredient> ingredients) {
      */
     public Ingredient findIngredientByName(IngredientName ingredientName) {
         requireNonNull(ingredientName);
-        for (int i = 0; i < internalList.size() - 1; i++) {
+        for (int i = 0; i < internalList.size(); i++) {
```
Comment (Diwu-Yi) [inline src/main/java/seedu/address/model/ingredient/UniqueIngredientList.java:71]: I don't think the -1 should be removed

**Review by Diwu-Yi — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/ingredient/UniqueIngredientList.java (hunk 1 above)
Comment (WM71811) [reply src/main/java/seedu/address/model/ingredient/UniqueIngredientList.java:71]: I think you referred to the wrong method, thanks for reviewing!

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Review by yanbingtao — APPROVED** [review]
LGTM

**Pull request merged**